test/e2e/lib/components/abstract-editor-component.js

```diff
@@ -34,10 +34,6 @@ export default class AbstractEditorComponent extends AsyncBaseContainer {
 	}
 
 	async openBlockInserterAndSearch( searchTerm ) {
-		await this.runInCanvas( async () => {
-			await driverHelper.scrollIntoView( this.driver, By.css( '.editor-styles-wrapper' ), 'start' );
-		} );
-
```

Comment:
Im not sure why this scroll step was originally necessary here, but things seem to work well without it!

Removing this seems fine if everything else seems to work without it..  Im realizing the problem here comes down to `runInCanvas` not actually limiting to the canvas in the post editor.  Since there are `.editor-styles-wrappers` outside the canvas (such as in pattern previews), those are being selected instead.